pyproject.toml

```diff
@@ -28,8 +28,9 @@ trace-updater = ">=0.0.8"
 numpy = ">=1.14"
 Flask-Cors = "^3.0.10"
 orjson = "^3.8.0"  # Faster json serialization
-Werkzeug = "2.1.2"  # Fixating werkzeug version because of:
+# TODO -> check other werkzeug versions: 2.1.2 and 2.1.1 seem to work.
```

Comment:
Should this be included in this PR? For me it's fine to move this to another PR

Reply:
I think it can be included!

Reply:
Will merge this PR and leave this for what it is